View/multiworldClient.py

```diff
@@ -1,29 +1,118 @@
 import asyncio
 
 from PySide6.QtWidgets import *
+from PySide6.QtCore import QThread, QObject, Signal, Slot
 
 from View.uiMultiworldClient import uiMultiworldClient
 from Model.serverConfig import ServerConfig
 from Model.setUpDto import SetUpDto
 import Client.clientCommunication as clientFunctions
 
+from Model.config import Config
+
+
+class JoinServerWorker(QObject):
+    
+    message = Signal(str) # to communicate with parent (gui) thread
+
+    def __init__(self) -> None:
+        super(JoinServerWorker, self).__init__()
+        self.config = Config.get_config()
+
+
+    def run(self) -> None:
+        print("Setting Fields")
+        server_config = ServerConfig(self.config._ServerAddress, self.config._Port,
+        self.config._World_id, 'admin', 'adminPass')
+        set_up_dto = SetUpDto(1, self.config._Game_Room, None, False) # 1 = world amount, just 1 for testing to match with number assigned to input box
+        try:
+            print("Into Listener")
+            asyncio.run(clientFunctions.start_connections(server_config, set_up_dto, self.message))
+            
+        except RuntimeWarning:
+            self.send_message("Failed to Create Room")
+
+    def send_message(self, msg: str) -> None: # to (maybe) receive messages from the actual connection things, haven't gotten that far yet
+        self.message.emit(msg)
 
 class MultiworldClientWindow(QMainWindow):
 
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
No need to clarify that the `__init__` returns None. As a constructor it has a clear purpose and usage already.